packages/compass-import-export/package.json

```diff
@@ -111,7 +111,6 @@
     "nyc": "^15.0.0",
     "peer-deps-externals-webpack-plugin": "^1.0.2",
     "postcss-loader": "^2.1.6",
-    "pre-commit": "^1.1.3",
```

Comment:
/cc @mcasimir I know that you were planning to clean it up in another PR, but it causes issues when using workspaces so I had to do this clean up here to be able to commit changes and I thought it would be easier to just do a clean up in the branch right away